README.md

```diff
@@ -22,6 +25,14 @@ Allows you to use a TFVC repository as an SCM in Jenkins jobs. At the moment, th
 
 The plugin will automatically create a workspace in TFS/Team Services and map a work folder (in the Jenkins workspace) to it.
 
+### Git
+The TFS plug-in for Jenkins enhances the Git plug-in for Jenkins by adding some integration features:
+* A push trigger, to request builds of specific commits in Git repositories without needing to schedule SCM polling
+* A build step that adds a "build pending" status to the associated pull request and/or commit in TFS/Team Services
+* A post-build action that add a "build completed" status to the associated pull request and/or commit in TFS/Team Services
+* Some endpoints for TFS/Team Services to use to activate the integration.
```

Comment:
As a user, do I need to know what are those "some endpoints" at all?  If yes, I'd appreciate a link to learn more about those endpoints.  If not really, this creates some confusion in my mind already...